src/lamp_py/performance_manager/alerts.py

```diff
@@ -20,16 +20,19 @@
 from lamp_py.postgres.metadata_schema import MetadataLog
 from lamp_py.postgres.postgres_utils import DatabaseManager
 from lamp_py.runtime_utils.process_logger import ProcessLogger
+from lamp_py.runtime_utils.remote_files import (
+    S3_PUBLIC,
+    LAMP,
+)
 
 from .gtfs_utils import BOSTON_TZ
 
 
 class AlertsS3Info:
     """S3 Constant info for Alerts Parquet File"""
 
-    bucket_name: str = os.environ.get("PUBLIC_ARCHIVE_BUCKET", "")
     s3_path: str = "s3://" + os.path.join(
-        bucket_name, "lamp", "tableau", "alerts", "LAMP_RT_ALERTS.parquet"
+        S3_PUBLIC, LAMP, "tableau", "alerts", "LAMP_RT_ALERTS.parquet"
```

Comment:
i think this should be another instance of the `S3Location` class rather than building up the path in this file.